setup.py

```diff
@@ -71,6 +71,10 @@
         "apache-beam[gcp]==2.26.0",
         "apache-beam==2.26.0",  # temporary for dataflow runner
         "google-apitools==0.5.31",  # temporary for dataflow runner
+        "jupyter-book==0.9.1",
+        "matplotlib==3.3.3",
+        "ghp-import==1.0.0",
+        "jedi==0.17.2",
```

Comment:
What is the need for these? :-) Do we need them in the setup.py?

Reply:
I got those by merging main into my branch, possibly due to the new docs changes